cli/src/cli_util.rs

```diff
@@ -2738,6 +2747,13 @@ fn handle_early_args(
     if args.no_pager.unwrap_or_default() {
         args.config_toml.push(r#"ui.paginate="never""#.to_owned());
     }
+    if let Some(key) = args.sign_with {
+        args.config_toml.push(format!(r#"signing.key="{key}""#));
```

Comment:
`key` is free-free form text, so quoting can fail. Can you add a TODO comment?

Or we can use `toml_edit` to do proper quoting: `toml_edit::Value::from(key)` or something. (untested)

Reply:
I added a TODO comment.